.re-natal

```diff
@@ -42,6 +42,7 @@
     "rn-snoopy/stream/buffer",
     "react-native/Libraries/vendor/emitter/EventEmitter",
     "react-native-background-timer",
+    "react-native-device-info",
```

Comment:
are you sure we need this lib? we had PR https://github.com/status-im/status-react/pull/2897/files#diff-02290a2c60d0fc8c53f1fccc76d23a96R42 with time localizations, maybe we could use here the same approach?

Reply:
It depends, so the use case would be:

I live in britain, so my locale is GB, but I find AM PM dreadful as I am Italian :), so I always set on my device 24hrs format.

If we don't want to provide for this use case, then yes we can just remove it and live that.

But also that code above is a bit buggy, it uses the timezone offset, which basically means that he does not take into consideration DST ( if you give it a timestamp of 0 now (in BST) it would mark the message as Jan 1, 01:00, which is not correct as during winter time we are in UTC).
Effectively once you move to DST all the messages sent in winter time will change their timestamp with a +1 hour.

(This is just an example, timezone are so complicated, between 68-71 Britain did not change to UTC, all year BST, but you get the idea :) )

So if we want to correct this we would probably have to get  the timezone from the device, I think.

Reply:
We usually try to avoid adding new dependencies (native even more). For complexity reasons but most importantly security reasons. If this only for that particular corner usage I think we could pass it for now. 
Suggested workaround: use italian locale! :D

Reply:
@jeluard happy to remove it for now, will likely be coming handy if we want to fix the issue mentioned above